Makefile

```diff
@@ -240,7 +240,7 @@ $(RELEASE_BUILD_DIR)/$(LIB_SYMLINK2): $(RELEASE_LIB)
 LIBDIR ?= usr/lib
 ABS_LIBDIR := $(shell echo /$(LIBDIR) | sed -r 's|/+|/|g')
 
-$(PKGCONFIG): $(LIB_NAME).pc.in Makefile
+$(PKGCONFIG): $(LIB_NAME).pc.in Makefile $(VERSION_FILE)
```

Comment:
Dependency on Makefile can be removed. Originally the version was hardcoded in the Makefile, and that's where that dependency comes from. And then when I switched to parsing the header file, I forgot to update it.